package.json

```diff
@@ -347,6 +348,7 @@
 		"stylelint": "9.10.1",
 		"supertest": "3.4.2",
 		"svgstore-cli": "1.3.1",
+		"typescript": "3.6.4",
```

Comment:
hah. how was typescript getting pulled in prior?

Reply:
It much have been calypso-build, which also seems to need an update. I'm not sure why we have TypeScript there as a dependency, I don't think we use it directly 🤔